<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(170, 255, 255);</string>
  </property>
  <widget class="QPushButton" name="reserve_tickets_button">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>180</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(147, 221, 221);</string>
   </property>
   <property name="text">
    <string>Забронировать билеты</string>
   </property>
  </widget>
  <widget class="QPushButton" name="search_passenger_tickets_button">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>250</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(147, 221, 221);</string>
   </property>
   <property name="text">
    <string>Найти билеты пассажира</string>
   </property>
  </widget>
  <widget class="QPushButton" name="search_buyed_race_tickets_button">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>320</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(147, 221, 221);</string>
   </property>
   <property name="text">
    <string>Найти проданные билеты рейса</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
